.github/workflows/event-nightly.yml

```diff
@@ -0,0 +1,12 @@
+name: nightly
+
+on:
+  schedule:
+    - cron: '0 20 * * *'
+
+jobs:
```

Comment:
I suggest still having the multi-build running every night. If we are not merging PRs for some time, it could indicate that something breaks in some flow (like we had now with rocky8....)